train.py

```diff
@@ -220,14 +228,15 @@
     resnet = resnet_dict[args.resnet](pretrained=True)
     model = model_class(resnet, **model_kwargs)
     model = model.cuda()
+    model = nn.DataParallel(model)
```

Comment:
What is the DataParallel wrapper for?

Reply:
Hey, sorry, I forgot to mention....I used DataParallel to train on multiple gpus...should work with a single gpu as well as it wraps over the model....

Reply:
I can remove it

Reply:
That's fine, what about the alterations to the model code?

Reply:
Pretty sure, no alterations required as it wraps it. I suggest to have a look at a wonderful tutorial by pytorch on nn.DataParallel; [Data Parallel Tutorial Pytorch](https://pytorch.org/tutorials/beginner/blitz/data_parallel_tutorial.html)

Reply:
Ohhh! I see the `.module` is because of the DataParallel wrapper, gotcha.